awx/api/serializers.py

```diff
@@ -2176,11 +2176,17 @@ class Meta:
 class InventoryUpdateSerializer(UnifiedJobSerializer, InventorySourceOptionsSerializer):
 
     custom_virtualenv = serializers.ReadOnlyField()
+    source_project = serializers.SerializerMethodField(
+        help_text=_('The project used for this job.'),
+    )
```

Comment:
As a method field, this won't be filterable or anything like that, which someone might expect.

Reply:
How can we make it filterable?

Reply:
You _can_ add the field and introduce a new migration for it. This also requires populating the field, which you would probably do at whatever point we already populate `source_project_update`. See:

https://github.com/ansible/awx/blob/51b51a9bf73de4a21c9aacb5c0c24f31bd44f04b/awx/main/tasks.py#L1703

You would add there `source_project=project_update.project`. I don't know _for sure_ right now that this is the only place that needs to be done.

Reply:
True, this might not be the best option for those reasons.